spynnaker/pyNN/models/neural_projections/projection_machine_edge.py

```diff
@@ -43,6 +43,26 @@ def filter_edge(self, graph_mapper):
                 post_hi = graph_mapper.get_slice(self.post_vertex).hi_atom
                 if pre_hi < post_lo or pre_lo > post_hi:
                     n_filtered += 1
+            elif isinstance(synapse_info.connector, FromListConnector):
+                pre_hi = graph_mapper.get_slice(self.pre_vertex).hi_atom
+                post_hi = graph_mapper.get_slice(self.post_vertex).hi_atom
+                pre_app_vertex = graph_mapper.get_application_vertex(
+                    self.pre_vertex)
+                post_app_vertex = graph_mapper.get_application_vertex(
+                    self.post_vertex)
+                pre_slices = graph_mapper.get_slices(pre_app_vertex)
+                post_slices = graph_mapper.get_slices(post_app_vertex)
+                # run through connection list and check for any connections
+                # between the pre and post vertices that could be filtered
+                try:
+                    synapse_info.connector._split_connections(
```

Comment:
See comment in the from_list_connector - once get_n_connections exists, you can just call this method.